src/jit/codegenlegacy.cpp

```diff
@@ -20853,22 +20868,23 @@ GenTreePtr Compiler::fgLegacyPerStatementLocalVarLiveness(GenTreePtr startNode,
                 // This ensures that the block->bbVarUse will contain
                 // the FrameRoot local var if is it a tracked variable.
 
-                if (tree->gtCall.IsUnmanaged() || (tree->gtCall.IsTailCall() && info.compCallUnmanaged))
-                {
-                    /* Get the TCB local and mark it as used */
+                if (!opts.ShouldUsePInvokeHelpers())
```

Comment:
Nit: It would be nice to have `{` / `}` around this. (I am surprised that the formatter does not complain about this.)